dtschema/lib.py

```diff
@@ -512,6 +512,15 @@ def fixup_node_props(schema):
         schema.setdefault('patternProperties', dict())
         schema['patternProperties']['pinctrl-[0-9]+'] = True
 
+    for key in keys:
+        if "assigned-clocks" in key:
+            break
+
+    else:
```

Comment:
Thanks for doing this.

We can do slightly better. Only add them if there is a clocks property.

Reply:
That's a good idea indeed, I've just pushed an updated version

Reply:
Humm, not seeing a new version. Anyways I updated it and committed it.